sdk/spring/spring-cloud-azure-resourcemanager/src/main/java/com/azure/spring/cloud/resourcemanager/implementation/crud/AbstractResourceCrud.java

```diff
@@ -99,11 +122,26 @@ public T getOrCreate(K key) {
         return create(key);
     }
 
+    @Override
+    public T getOrCreate(K key, @Nullable P properties) {
```

Comment:
If we have both `public T getOrCreate(K key)` and `public T getOrCreate(K key, @Nullable P properties)`, the second shouldn't accept a nullable proeprties.